### PR TITLE
histogram: calculate lengths using round-to-nearest

### PR DESCRIPTION
**Pull request by siadat**
It was rounding down.

### CODE REVIEW

File: requester/print.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func (r *report) printHistogram() {
 		// Normalize bar lengths.
 		var barLen int
 		if max > 0 {
-			barLen = counts[i] * 40 / max
+			barLen = (counts[i]*40 + max/2) / max
```
Comment (mdakin) [inline requester/print.go]: LGTM, but maybe just add 0.5 instead.

Comment (siadat) [inline requester/print.go]: Do you mean converting to a float, add 0.5, and convert back to an int?

Comment (mdakin) [inline requester/print.go]: Ah nevermind my comment, your version is good.

**Comment by rakyll** [comment]
Merged, thanks!

**Pull request merged**